.appveyor/build_libedit.bat

```diff
@@ -75,22 +75,37 @@ if not exist "%BUILDDIR%\wineditline-%LIBEDIT_VERSION%" (
 2>nul rd /S /Q "%BUILDDIR%\wineditline-%LIBEDIT_VERSION%\include"
 mkdir "%BUILDDIR%\wineditline-%LIBEDIT_VERSION%\build"
 
-"%MSYSDIR%\bin\bash" --login -i -c "cd \"$BUILDDIR_M/wineditline-$LIBEDIT_VERSION/build\" && $CMAKE_BIN -DLIB_SUFFIX=32 -G \"MSYS Makefiles\" .."
+if exist "%VS90COMNTOOLS%..\..\VC\bin\vcvars32.bat" (
```

Comment:
I think we should add this check in set_paths.bat instead here and if "%VS90COMNTOOLS%vcvars32.bat" doesn't exists then exit from set_paths.bat itself...

Also any specific reason to use "%VS90COMNTOOLS%..\\..\VC\bin\vcvars32.bat" instead "%VS90COMNTOOLS%vcvars32.bat"

Reply:
Thanks, I applied the changes you suggested.